wsic/utils.py

```diff
@@ -13,46 +13,31 @@ def downsample_shape(
     downsample: int,
     rounding_func: Callable[[Number], int] = floor,
 ) -> Tuple[int, ...]:
-    r"""Calculate the shape of an array after downsampling by a factor.
-
+    """Calculate the shape of an array after downsampling by a factor.
     The shape is calculated by dividing the shape of the baseline array
     by the downsample factor. The output is rounded to the nearest
     integer using the provided rounding function. E.g. for a founding
     function of `floor`, the following opertion is performed
     :math:`\lfloor \frac{shape}{downsample} \rfloor`. If a channel
     dimension is specified, the dimension is left unchanged.
-
```

Comment:
Can you please leave the spaces between blocks in the docstrings. Removing all spacing makes them hard to read.